cmd/microservice-lootbox-reward-top-winners/main.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2023 Fluidity Money. All rights reserved. Use of this
+// source code is governed by a GPL-style license that can be found in the
+// LICENSE.md file.
+
+package main
+
+import (
+	"time"
+
+	"github.com/fluidity-money/fluidity-app/lib/databases/timescale/lootboxes"
+	"github.com/fluidity-money/fluidity-app/lib/log"
+	"github.com/fluidity-money/fluidity-app/lib/util"
+)
+
+const (
+    // layoutISO to parse the date in the given formta
+    layoutISO = "2006-01-02"
+    // EnvRewardDate to override the date to reward top winners for, otherwise using the current date
+    // In the format yyyy-mm-dd, parsed in Australia/Adelaide (ACST)
+    EnvRewardDate = `FLU_REWARD_DATE`
+)
+
+// runs as a cron service, every day at 00:00:05 (adelaide time)
+// assumes there are at least 5 active users in a given day
+func main() {
+    location, err := time.LoadLocation("Australia/Adelaide")
```

Comment:
Should be done after the env

Reply:
Also, could make Austraia/Adelaide a constant, or an environment variable argument